canister/candid.did

```diff
@@ -52,6 +52,9 @@ type fees = record {
   get_current_fee_percentiles_maximum : nat;
   send_transaction_base : nat;
   send_transaction_per_byte : nat;
+  get_block_headers_base : nat;
+  get_block_headers_cycles_per_ten_instructions : nat;
+  get_block_headers_maximum : nat;
```

Comment:
Here we're assuming that we will charge based on the number of instructions it takes to process the request. That's not necessarily how we want to charge. We may prefer for instance to charge based on the number of headers retrieved. I'd suggest removing anything related to fees from this PR and we can add it once we know how we want to charge.

How we want to charge can be a separate discussion with @THLO.

Reply:
Done.